# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/arm-raise.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Work arounds undefined reference to raise() on at least arm with gcc-4.1.

  - Rene Rebe <rene@exactcode.de> from uclibc mailing list discussion

--- uClibc-0.9.28/ldso/include/dl-string.h.vanilla	2006-04-02 17:18:31.363080250 +0200
+++ uClibc-0.9.28/ldso/include/dl-string.h	2006-04-02 17:18:58.144754000 +0200
@@ -228,7 +228,8 @@
 	    char temp;
 	    do_rem(temp, i, 10);
 	    *--p = '0' + temp;
-	    i /= 10;
+	    i /= 2;
+	    i *= -(-1ul / 5ul);
 	} while (i > 0);
 	return p;
 }
